source/winAPI/sessionTracking.py

```diff
@@ -231,145 +210,41 @@ def isLockStateSuccessfullyTracked() -> bool:
 	I.E. Registered for session tracking AND initial value set correctly.
 	@return: True when successfully tracked.
 	"""
-	return (
-		not _sessionQueryLockStateHasBeenUnknown
-		or not _isSessionTrackingRegistered
+	# TODO: improve deprecation practice on beta/master merges
+	log.error(
+		"NVDA no longer registers session tracking notifications. "
+		"This function is deprecated, for removal in 2023.1. "
+		"It was never expected that add-on authors would use this function"
 	)
+	return _currentSessionState and not _currentSessionState._windowsLockStateIsUnknown
 
 
-def register(handle: HWND) -> bool:
-	"""
-	@param handle: handle for NVDA message window.
-	When registered, Windows Messages related to session event changes will be
-	sent to the message window.
-	@returns: True is session tracking is successfully registered.
-
-	Blocks until Windows accepts session tracking registration.
-
-	Every call to this function must be paired with a call to unregister.
-
-	If registration fails, NVDA may not work properly until the session can be registered in a new instance.
-	NVDA will not know when the lock screen is activated, which means it becomes a security risk.
-	NVDA should warn the user if registering the session notification fails.
-
-	https://docs.microsoft.com/en-us/windows/win32/api/wtsapi32/nf-wtsapi32-wtsregistersessionnotification
-	"""
-
-	# OpenEvent handle must be closed with CloseHandle.
-	eventObjectHandle: HANDLE = ctypes.windll.kernel32.OpenEventW(
-		# Blocks until WTS session tracking can be registered.
-		# Windows needs time for the WTS session tracking service to initialize.
-		# NVDA must ensure that the WTS session tracking service is ready before trying to register
-		SYNCHRONIZE,  # DWORD dwDesiredAccess
-		False,  # BOOL bInheritHandle - NVDA sub-processes do not need to inherit this handle
-		# According to the docs, when the Global\TermSrvReadyEvent global event is set,
-		# all dependent services have started and WTSRegisterSessionNotification can be successfully called.
-		# https://docs.microsoft.com/en-us/windows/win32/api/wtsapi32/nf-wtsapi32-wtsregistersessionnotification#remarks
-		"Global\\TermSrvReadyEvent"  # LPCWSTR lpName - The name of the event object.
+def register(handle: int) -> bool:
+	# TODO: improve deprecation practice on beta/master merges

```

Comment:
I'm not sure what this means. Happy for the plan to be documented on the PR description.